src/osx/cocoa/utils_base.mm

```diff
@@ -25,7 +25,7 @@
 // our OS version is the same in non GUI and GUI cases
 wxOperatingSystemId wxGetOsVersion(int *majorVsn, int *minorVsn)
 {
-#if MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_10
+#if MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_10 || defined(__WXOSX_IPHONE__)
```

Comment:
For me this results in compilation failures, probably due to too old SDK (7.0). Is there some way to test for this?